index.html

```diff
@@ -387,6 +371,49 @@ <h2>
         applied.
       </aside>
     </section>
+    <section>
+      <h2>
+        Calculating an accessible name
+      </h2>
+      <p>
+        The steps for <dfn data-lt="calculate an accessible name">calculating an
+        accessible name</dfn> are given by the following algorithm. The
+        algorithm takes an [=image resource=] |input| and an optional
+        [=Response=] |image| which is the result of [=fetching an image
+        resource=]. It returns a [=string=].
+      </p>
+      <ol>
+        <li>If |input|["label"] is a [=string=]:
+          <ol>
+            <li>Return |input|["label"].
+            </li>
+          </ol>
+        </li>
+        <li>Otherwise if an [=external label=] can be substituted:
+          <ol>
+            <li>Return the relevant [=external label=].
+            </li>
+          </ol>
+        </li>
+        <li>Otherwise if |image| provides an [=embedded accessible name=]:
+          <ol>
+            <li>Return the [=embedded accessible name=].
+            </li>
+          </ol>
+        </li>
+        <li>Otherwise:
+          <ol>
+            <li>Return an empty [=string=] ("").
+            </li>
+          </ol>
+        </li>
+      </ol>
+      <aside class="note">
+        Implementors may delay [=fetching an image resource=] in order to
```

Comment:
This is a conformance requirement, so should go where the algorithm is.

Reply:
Do you think this is just redundant with [the initial description of `label` which contains the MAY language](https://github.com/w3c/image-resource/blob/03ac360e303d34dfb965eeaa6cff3f69e6efe5e9/index.html#L227)?

Reply:
In retrospect, I think it is redundant.